src/Umbraco.Web.UI/config/imageprocessor/processing.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<processing preserveExifMetaData="true" fixGamma="false" interceptAllRequests="false" allowCacheBuster="true">
+<processing preserveExifMetaData="false" fixGamma="false" interceptAllRequests="false" allowCacheBuster="true">
```

Comment:
We should bump Umbraco to depend on the very latest ImageProcessor.Web then we can do (there are probably some transforms somewhere)
`preserveExifMetaData="true" metaDataMode="Copyright"`
See https://github.com/JimBobSquarePants/ImageProcessor/pull/660 for more info

Reply:
Problem is we cannot depend on the latest because it has a minimum target of net452 and we are net45, not sure if james is using specific net452 APIs (doubt it), else it would be better to just allow net45 as the minimum

Reply:
Yeah, I can't do that. NET 4.5.2 is the minimum supported framework by Microsoft.

I would advise you actually make the bump here to 4.5.2 as there are now potential security issues that will remain unpatched.

Reply:
Yup but unfortunately that's a breaking change to bump our nuget minimum to that version, that would break packages targeting 4.5.0 (same scenario as this actually). Hooray for backwards compat.

Reply:
Is that specific to 4.5.0 and 4.5.2?

Reply:
The way Nuget works is that you have to advertise what your package can run under. In this case the PostProcessor package advertises that it only supports a minimum version of .net 452, (though it will still work under 450 unless specific 452 APIs are in use). Because our nuget package advertises that it's minimum is 450, nuget will not allow a reference to the PostProcessor package. 

Similarly, if we were to update our nuget package to be a min of 452, then any package developer referencing our project that has a min of 450 would have to bump their nuget, release a new major version, etc...

Reply:
What might be possible is if we bump our runtime in all projects to 452 and then we can reference this package but we don't adjust our nuspec which will still support 450

Reply:
As far as I understood it, and experienced it to my knowledge, with Nuget It's perfectly possible to reference and install a package that targets 4.5.0 into a 4.5.2+ targeting package. Only the reverse is prohibited.

Reply:
Either way, I don't think this package should be bundled with Umbraco. It's gonna make the move to .NET Core difficult in the future.

Reply:
> t's perfectly possible to reference and install a package that targets 4.5.0 into a 4.5.2+ targeting package. Only the reverse is prohibited.

Exactly :)  Umbraco targets 4.5.0 and this package targets 4.5.2 so Umbraco cannot install this package

All good with me on not bundling this package, i made this PR as a demo on how to create a PR which was based on a community member's request.

Reply:
@Shazwazza so this one can be closed ?

Dave